server/auth/routes.js

```diff
@@ -2,53 +2,89 @@ const express = require("express");
 const router = express.Router();
 const jwt = require("jsonwebtoken");
 const db = require("../services/database/users");
+var MyMethods = require("./validator.js");
```

Comment:
What is `MyMethods`? Try to use explicit names, in this case a better name would be `Validators`. Also try not to use `var` which is old javascript, instead use either `const` or `let`. A better way to import all the validator functions can be: `const { validateEmail, validateName, validatePassword, validateRole } = require('./validator');`

Reply:
I removed the name and changed to const